pyaedt/modules/monitor_icepak.py

```diff
@@ -517,18 +517,17 @@ def _delete_removed_monitors(self):
 
     @pyaedt_function_handler()
     def insert_monitor_object_from_dict(self, monitor_dict, mode=0):
-        """
-        Insert a monitor.
+        """Insert a monitor.
 
         Parameters
         ----------
         monitor_dict : dict
            Dictionary containing monitor object information.
         mode : int
-            Integer to select the information to handle. To identify faces, vertices, surfaces and object to which the
-            monitor is to be assigned it is possible to use:
-                ids and names, mode=0, required dict keys: "Name", "Type", "ID", "Quantity".
-                positions, mode=1, required dict keys: "Name", "Type", "Geometry Assignment", "Location", "Quantity".
+            Integer to select the information to handle. To identify the faces, vertices, surfaces, and object to which to
+            assign the monitor to, you can use:
```

Comment:
@gmalinve This should fix the pre-commit issue.
```suggestion
            Integer to select the information to handle. To identify the faces, vertices,
            surfaces, and object to which to assign the monitor to, you can use:
```